community/planning/features.md

```diff
@@ -22,4 +22,4 @@ community meeting.
 | [High & Low-level Transaction REST API]({% link
 community/planning/high_low_level_transaction_rest_api.md %}) | An API for submitting higher-level JSON transactions |
 | [Product Sapling]({% link community/planning/product_sapling.md %}) | A front-end for Grid Product functionality |
-| [Griddle]({% link community/planning/griddle/index.md %}) | A REST API for transacting with Grid |
+| [Future REST API]({% link community/planning/rest_api/index.md %}) | A proposed future REST API for transacting with Grid |
```

Comment:
Should this link go in the "High & Low-level transaction rest api" maybe, rather than removing griddle's entry ?

Reply:
Nvm, realized this is an entry we added specifically for this api doc